src/communication/message-factory.ts

```diff
@@ -154,7 +156,7 @@ export abstract class MessageFactory {
       messageResponseId: message.messageId,
       serialize: serialization,
       content: response instanceof Error ? null : prepare(),
-      error: response instanceof Error ? prepare() : null
+      error: response instanceof Error ? prepareError(response) : null
     });
```

Comment:
This is not part of the @Output() change but it is important nonetheless. If an uncaught exception happens, it doesn't get serialized properly in some instances